src/wallet/rpcwallet.cpp

```diff
@@ -1151,9 +1151,12 @@ UniValue getbalance(const UniValue& params, bool fHelp)
     if (fHelp || params.size() > 5)
         throw runtime_error(
             "getbalance ( \"(dummy)\" minconf includeWatchonly inZat asOfHeight )\n"
-            "\nReturns the wallet's available transparent balance. This total\n"
-            "currently includes transparent balances associated with unified\n"
-            "accounts. Prefer to use `z_getbalanceforaccount` instead.\n"
+            "\nReturns the wallet's available balance associated with legacy\n"
+            "transparent addresses. Prefer to use `z_getbalanceforaccount` when\n"
+            "possible.\n"
```

Comment:
`z_getbalanceforaccount` isn't a replacement for this method. As far as I understand, there will be no replacement until #6356 is implemented [edit: except maybe `getwalletinfo`?]. In any case the wording is misleading because there is *no* case in which `z_getbalanceforaccount` can replace `getbalance` after this PR.

(Just realised I suggested the current wording, sorry.)

```suggestion
            "transparent addresses.\n"
```